crates/ruff/src/rules/flake8_simplify/rules/fix_if.rs

```diff
@@ -45,7 +45,7 @@ pub(crate) fn fix_nested_if_statements(
 
     // If this is an `elif`, we have to remove the `elif` keyword for now. (We'll
     // restore the `el` later on.)
-    let is_elif = contents.starts_with("elif");
+    let is_elif = contents.trim_start().starts_with("elif");
```

Comment:
@konstin - Can we avoid this now, after the elif refactor?

Reply:
yep, the following patch avoid looking into the string for the elif: https://gist.github.com/konstin/4baa8825689d7e41ae74d0a4b796f4a5